foundations/javascript_basics/project_calculator.md

```diff
@@ -26,11 +26,15 @@ Here are some use cases (abilities your project needs to have):
    1. You should already have the code that can populate the display, so once `operate()` has been called, update the display with the 'solution' to the operation.
    2. This is the hardest part of the project. You need to figure out how to store all the values and call the operate function with them. Don't feel bad if it takes you a while to figure out the logic.
 6. Gotchas: watch out for and fix these bugs if they show up in your code:
-   1. Users should be able to string together several operations and get the right answer, with each pair of numbers being evaluated at a time. For example, `12 + 7 - 5 * 3 =` should yield `42`. An example of the behavior we're looking for would be [this student solution](https://mrbuddh4.github.io/calculator/). **Your calculator should not evaluate more than a single pair of numbers at a time. If you enter a number then an operator and another number that calculation should be displayed if your next input is an operator. The result of the calculation should be used as the first number in your new calculation.**
-   2. You should round answers with long decimals so that they don't overflow the screen.
-   3. Pressing `=` before entering all of the numbers or an operator could cause problems!
-   4. Pressing "clear" should wipe out any existing data.. make sure the user is really starting fresh after pressing "clear"
-   5. Display a snarky error message if the user tries to divide by 0... don't let it crash your calculator!
+   1. Users should be able to string together several operations and get the right answer, with each pair of numbers being evaluated at a time. For example, `12 + 7 - 5 * 3 =` should yield `42`. An example of the behavior we're looking for would be [this student solution](https://mrbuddh4.github.io/calculator/).
+   2. **Your calculator should not evaluate more than a single pair of numbers at a time. Here is an example of the correct behavior:**
+      1. **A user enters a number, an operator, and a second number. They should see the second number at this point**, e.g. `12 + 7` displays `7`.
+      2. **Next, the user enters a second operator. They should immediately see the result of the calculation from step 1**, e.g. `12 + 7 -` displays `19`.
+      3. **The result from step 2 should then be used as the first number in the user's next calculation**, e.g. `12 + 7 - 5` displays `14`. **The process has now started over.**
```

Comment:
Step 3 should indeed say "displays `5`" - thanks for the correction.

The idea behind this addition to the instructions was to show the display value after subsequent user inputs, not to show the underlying calculations as they go, which was why I went with 12 + 7 - 5 not 19 - 5.

Regardless, if you find it confusing then feel free not to use it. Thanks!

Reply:
If we want to try to update the verbiage, instead of nesting a third list with more steps, it might be simpler to simply replace line 29 with:

```
   1. Users should be able to string together several operations and get the right answer, with each pair of numbers being evaluated at a time. For example, `12 + 7 - 5 * 3 =` should yield `42`. An example of the behavior we're looking for would be [this student solution](https://mrbuddh4.github.io/calculator/). 

        **Your calculator should not evaluate more than a single pair of numbers at a time.** If you press a number button (`12`), followed by an operator button (`+`), a second number button (`7`), and finally a second operator button (`-`), your calculator should first evaluate the first pair of numbers (`12 + 7`). Then, the result of that calculation (`19`) should be displayed and used as the first number in your new calculation using your other operator (`-`).
```

Having a third nested list I feel is a little too much, and I think the same point can be put across with something more inline like the above. I'm fine with leaving things as is as well, as I can't remember if these instructions were found to be confusing by a significant number of users (not to say that isn't the case, I just can't recall).

Reply:
I like your inline fix (replacing line 29 as you have). I agree that it would be better than a third nested list. Feel free to change it that way if you want, or leave it as is. I proposed a change because I noticed what I thought was a good number of requests and questions about the topic of order of operations with the calculator project. But others may not have noticed this; it was just my impression.

Reply:
I think we should go with something like that for sure. This does need to be updated regardless.

Reply:
@msespos Definitely trust your judgement on this needing an update, it's just been a while since I've personally seen people talking about the project in chat so couldn't really comment in that regard.

Would you want to push a new commit with something more inline? Either copy+pasting the example above or if you have an idea definitely feel free to put your voice into it.

Reply:
OK I edited to use @thatblindgeye 's fix, with a slight change in wording, but keeping it basically the same length and  content, and inline. Hopefully this works for everyone!

Reply:
Looks good! Thanks for bringing up the issue in the first place and being so accomodating to and making the requsted changes.